File: src/main/java/com/jason/guess/game.java
package com.jason.guess;

import java.util.Random;
import java.util.Scanner;

public class game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10) + 1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret) {
            System.out.println("Please enter a number: ");
            number = scanner.nextInt();
            System.out.println(number);
            if(number < secret) {
                System.out.println("Higher");
            } else if(number > secret){
                System.out.println("Lower");
            } else {
                System.out.println("Great, the number is "+ number);
            }
        }

        /*int i = 0;
        do {
            System.out.println(i);
            i++;
        } while (  i < 3);*/
    }
}

File: src/main/kotlin/com/jason/Game.kt
package com.jason

import java.util.*

fun main() {
    val secret = Random().nextInt(10) + 1;
    println(secret)
//    val scanner = Scanner(System.`in`)
    var number = 0
    while (number != secret) {
        print("Please enter a number: ")
//        number = scanner.nextInt()
        number = readLine()!!.toInt()
        println(
            if(number < secret)
                "Higher"
            else if(number > secret)
                "Lower"
            else
                "Great, the number is $secret"
        )
    }
}